### PR TITLE
Packages/py recommonmark

### PR DESCRIPTION
**Pull request by Bidibulke**
Adding recipe for `recommonmark` (a python package)

### CODE REVIEW

File: lib/spack/docs/basic_usage.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+.. Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```
Comment (alalazo) [inline lib/spack/docs/basic_usage.rst]: Please revert this change

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-recommonmark/package.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    http://recommonmark.readthedocs.org"""
+
+    homepage = "https://github.com/readthedocs/recommonmark"
+    url      = "https://files.pythonhosted.org/packages/f5/71/046160d730f664662d65b3f8b399b519ad378ffa4369ff3b6060cf1318d7/recommonmark-0.6.0.tar.gz"
+
+    version('0.6.0', sha256='29cd4faeb6c5268c633634f2d69aef9431e0f4d347f90659fd0aab20e541efeb')
+
+    depends_on('py-commonmark')
+    depends_on('py-docutils')
+    depends_on('py-sphinx')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-recommonmark/package.py]: Can you also encode the version requirements from [here](https://github.com/readthedocs/recommonmark/blob/5ec3751ed8aae194cffd84bde2e4e33e1bd5182b/setup.py#L25-L28)?

**Review by alalazo — CHANGES_REQUESTED** [review]
LGTM, just a minor request.

File: var/spack/repos/builtin/packages/py-recommonmark/package.py (hunk 2 above)
Comment (Bidibulke) [reply var/spack/repos/builtin/packages/py-recommonmark/package.py]: Done

**Review by alalazo — APPROVED** [review]
